openshift_cli_installer/libs/managed_clusters/osd_clusters.py

```diff
@@ -22,6 +22,8 @@ def osd_create_cluster(cluster_data, must_gather_output_dir=None):
         client=cluster_data["ocm-client"],
         name=cluster_name,
     )
+    cluster_data["cluster-object"] = cluster_object
```

Comment:
We need to set `cluster_data["cluster-object"]` in the same place we do `cluster_data["ocm-client"]` `openshift_cli_installer.utils.cli_utils.prepare_clusters`